### DISCUSSION TITLE
Query Network Drive files

### DISCUSSION THREAD

**Discussion by kumarvvin** [prestodb/presto#15248] [General]
Since Presto can be used to Query data on S3, can we similarly use it for querying files on onprem network drives or AWS EFS ? Currently we are  crawling the files metadata into a database and using it for reporting, if we can use presto and directly query then we can simplify our solution and search and discovery of files on network drive .

**Comment by tooptoop4** [comment]
https://github.com/ebyhr/presto-flex or https://openlookeng.io/docs/docs/connector/localfile.html / https://github.com/openlookeng/hetu-core/tree/master/hetu-filesystem-client/src/main/java/io/hetu/core/filesystem
